packages/passport/sdk/src/Passport.ts

```diff
@@ -137,15 +138,28 @@ export class Passport {
     return this.passportImxProviderFactory.getProvider();
   }
 
-  public connectEvm(): Provider {
-    return new ZkEvmProvider({
+  public connectEvm(options: {
+    announceProvider: boolean
+  } = {
+    announceProvider: true,
+  }): Provider {
```

Comment:
Can we add a type for this please? Otherwise consumers may have to add their own

Reply:
@haydenfowler I can, although I was attempting to match the current approach in that class with  the same inline typing as `login` and `loginWithDeviceFlow` method. Do you want me to keep that consistent or add a new type for this method?